src/database/queries/reviewsResources.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+const ReviewResource = require('../models/index').ReviewResource
+
+exports.findAll = async (filter, attributes = {}) => {
+	try {
+		const res = await ReviewResource.findAll({
+			where: filter,
+			attributes,
+			raw: true,
+		})
+
+		return res
+	} catch (error) {
+		return error
+	}
+}
+exports.countDistinct = async (filter, attributes = {}) => {
+	try {
+		const res = await ReviewResource.count({
```

Comment:
@adithyadinesh0412 this function is not using anywhere. Please remove it if not required